migration/migrator/migrations/master/20210517142134_course_column_email.py

```diff
@@ -0,0 +1,29 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    sql = "ALTER TABLE emails ADD COLUMN IF NOT EXISTS course VARCHAR DEFAULT NULL;"
+    database.execute(sql)
+    
+    sql = "ALTER TABLE emails ADD COLUMN IF NOT EXISTS semester VARCHAR DEFAULT NULL;"
+    database.execute(sql)
```

Comment:
We should strive to be consistent with the ordering of these variables, where we always have `$semester` followed by `$course` (such as in file paths, routes, parameters, etc.).

A very minor nitpick, but strong internal consistency makes it things easier for other developers and in jumping around the codebase.

Reply:
Is there a better way to do this migration? I'm not sure how this could end up being out of order in the database.

Reply:
For the migration, it would be just doing `ADD COLUMN semester` before doing `ADD COLUMN course`. However, there's a number of places (e.g. `getEmailStatusWithCourse`) where you're doing `$course, $semester` and that should be `$semester, $course`. Again, it's more of a consistency that the order of these variables is always semester and then course.